inference_lib/src/aqlm/inference_kernels/cuda_kernel.cpp

```diff
@@ -103,6 +132,124 @@ torch::Tensor code1x16_matmat(
   return output;
 }
 
+torch::Tensor code1x16_dequant(
+  const torch::Tensor& codes,
+  const torch::Tensor& codebooks,
+  const torch::Tensor& scales
+) {
+  check_use_bfloat16(codebooks);
+  auto in_features = codes.size(1) * 8;
+  auto out_features = scales.size(0);
+
+  auto weight = torch::zeros({out_features, in_features},
```

Comment:
`torch::empty` should be faster than `zeros`.  Why zeros?

Reply:
I used it for debug to easily spot unmodified memory. Forgot to change it to `empty`. Thanks!